packages/components/src/mobile/color-settings/utils.native.js

```diff
@@ -0,0 +1,5 @@
+export const colorsUtils = {
+	subsheets: [ 'Settings', 'Color' ],
```

Comment:
I was thinking more of something like this:

```suggestion
	subsheets: {
		settings: 'Settings',
		color: 'Color',
	},
```

So we could use 

`colorsUtils.subsheets.color` instead of `colorsUtils.subsheets[ 1 ]` 

What do you think?